Tests/Article.Tests.ps1

```diff
@@ -10,7 +10,7 @@ set-location -Path $RootFolder.FullName
 
 Write-Verbose "Importing module"
 
-import-module .\PSHTML -Force
+import-module .\pshtml.psd1 -Force
```

Comment:
Same here:

As mentionned in my main comment, this will break, once we restructure the folder layout.